peas/tests/test_sensors.py

```diff
@@ -65,3 +79,55 @@ def test_create_telemetry(serial_handlers):
     assert report_num <= 2
     ser.disconnect()
     assert ser.is_connected is False
+
+# --------------------------------------------------------------------------------------------------
+
+def test_auto_detect_port_fail():
+    """Auto-detect will fail because loop:// handler doesn't print anything."""
+    #pytest.set_trace()
+    assert sensors_module.auto_detect_port('loop://') is None
+
+
+def test_no_handler_installed():
+    """Can't find our simulator, so fails."""
+    assert sensors_module.auto_detect_port('arduinosimulator://?board=telemetry') is None
```

Comment:
Sorry, why is this one `None`? Seems same as below that work.

Reply:
Comments updated to explain.